pkg/services/pluginsintegration/pluginsintegration.go

```diff
@@ -52,7 +52,7 @@ var WireSet = wire.NewSet(
 	coreplugin.ProvideCoreRegistry,
 	pluginscdn.ProvideService,
 	assetpath.ProvideService,
-	angulardetector.ProvideService,
+	angulardetector.ProvideInspector,
```

Comment:
Related to @andresmgot comment about splitting up detection and inspection in separate packages becomes clear  why that would be a good idea with this example.